examples/framework.rs

```diff
@@ -168,7 +168,7 @@ async fn setup<E: Example>(title: &str) -> Setup {
                 label: None,
                 features: (optional_features & adapter_features) | required_features,
                 limits: needed_limits,
-                shader_validation: true,
+                shader_validation: false,
```

Comment:
I managed to get Naga to a point where it can handle the skybox example when ported to WGSL, so we'll be able to turn this back on with this upcoming change

Reply:
fwiw, I didn't intend on committing this, it just got caught up in my force pushing nonsense